File: inutile/xx.py
import json
import random

def generate_data(filename):
    data = [{"id": i, "value": random.randint(0, 99)} for i in range(10)]
    with open(filename, 'w') as f:
        json.dump(data, f, indent=4)
    print(f"Data generated and saved to {filename}")

if __name__ == "__main__":
    generate_data("data.json")

File: inutile/xx.lua
local json = require("dkjson")

math.randomseed(os.time())

local data = {}
for i = 0, 9 do
    table.insert(data, {id = i, value = math.random(0, 99)})
end

local json_data = json.encode(data, {indent = true})

local file = io.open("data.json", "w")
file:write(json_data)
file:close()

print("Data generated and saved to data.json")